src/pages/address/fragments/transactions/fragments/AddressTransactionMobileRow.tsx

```diff
@@ -12,6 +12,27 @@ type Props = {
   network: string
 }
 
+type AddressProps = {
+  address: string
+  chain: string
+  network: string
+}
+
+const Address: React.FC<AddressProps> = ({ address, chain, network }) => {
+  const isAddress = address !== 'mint' && address !== 'burn'
+
+  return isAddress ? (
+    <Link
+      className="hash"
+      to={`${ROUTES.WALLET.url}/${chain}/${network}/${address}`}
+    >
+      <span className="text-primary">{truncateHash(address, true)}</span>
+    </Link>
+  ) : (
+    <span className="text">{address}</span>
+  )
+}
```

Comment:
I believe it can be a separate component and reused elsewhere.
Could pass params for the different items